crates/biome_js_analyze/src/lint/nursery/no_static_element_interactions.rs

```diff
@@ -0,0 +1,248 @@
+use crate::services::aria::Aria;
+use biome_analyze::context::RuleContext;
+use biome_analyze::{declare_rule, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_js_syntax::jsx_ext::AnyJsxElement;
+use biome_rowan::AstNode;
+use rustc_hash::FxHashMap;
+
+declare_rule! {
+    /// Enforce that non-interactive, visible elements (such as `<div>`) that have click handlers use the role attribute.
+    ///
```

Comment:
The explanation should be more descriptive, and we can follow the original jsx-a11y rule. We should add source link.

You can refer this previous PR: https://github.com/biomejs/biome/pull/656/files

Reply:
@unvalley 
fixed in this https://github.com/biomejs/biome/pull/2981/commits/ae8d3c7cfc1618b519f975974a78526bfab2adfd
I'm not very good at English, so again you may point this out to me...